label_errors.py

```diff
@@ -110,19 +87,28 @@ def download_assets(project_id, assets, api_key, data_path, job_name):
 )
 @click.option("--project-id", default=None, required=True, help="Kili project ID")
 @click.option("--verbose", default=0, type=int, help="Verbose level")
+@click.option(
+    "--disable-wandb",
+    default=False,
```

Comment:
default = True for this one?

Reply:
hum, maybe for label error we can put it to false